### PR TITLE
Archive and Undo added

### PR DESCRIPTION
**Pull request by armantaherian**
I have added a new section as “Done”, and an “Undo” button to the items.
I hope to add a general button to show/hide the section in future.

### CODE REVIEW

**Comment by cassidoo** [comment]
This looks great, thanks so much!

I have one really nitpicky request: can you make the undo arrow have the same thickness as the other buttons?  The line before the arrowhead is a little thinner.

File: views/components/Item.jsx [hunk 1]
```diff
@@ -4,21 +4,29 @@ import React from 'react';
...
-    if (!this.props.paused) {
+    if (!this.props.paused && !this.props.completed) {
       return (
         <div className="buttons">
           <button className="delete" onClick={() => this.props.onDelete(this.props.item)}></button>
           <button className="pause" onClick={() => this.props.onPause(this.props.item)}></button>
           <button className="complete" onClick={() => this.props.onComplete(this.props.item)}></button>
         </div>
       );
+    } else
```
Comment (cassidoo) [inline views/components/Item.jsx]: `} else if {` all on one line

File: views/components/ItemList.jsx [hunk 2]
```diff
@@ -35,6 +36,13 @@ class ItemList extends React.Component {
     this.props.updateItem(completedItem);
   }
 
+  undoItem(item) {
+    const undoedItem = Object.assign({}, item, {
```
Comment (cassidoo) [inline views/components/ItemList.jsx]: `undoneItem` might be better than `undoedItem` here ;)

**Comment by armantaherian** [comment]
Sounds good. :)
All the changes applied.

**Pull request closed without merging**